cluster/gce/config-default.sh

```diff
@@ -566,3 +573,18 @@ export WINDOWS_ENABLE_DSR="${WINDOWS_ENABLE_DSR:-false}"
 # TLS_CIPHER_SUITES defines cipher suites allowed to be used by kube-apiserver. 
 # If this variable is unset or empty, kube-apiserver will allow its default set of cipher suites.
 export TLS_CIPHER_SUITES=""
+
+# Optional: URL to download antrea-cni.exe for Windows node
+export WINDOWS_ANTREA_CNI_BINARY_URL="${WINDOWS_ANTREA_CNI_BINARY_URL:-https://github.com/vmware-tanzu/antrea/releases/download/v0.13.1/antrea-cni-windows-x86_64.exe}"
```

Comment:
Do we want CNI and Agent at the same version? If so it may make sense to use a common version or URL base.

Reply:
Most likely, but I am not assuming that. It could be different `.gke-x` versions in the future, so I would vote keep the flexibility there.